src/backend/joanie/tests/core/test_api_enrollment.py

```diff
@@ -779,6 +785,18 @@ def test_api_enrollment_read_detail_authenticated_owner_certificate(self):
                     "certificate_id": str(certificate.id),
                     "product_id": str(product.id),
                     "state": "draft",
+                    "payment_schedule": [
+                        {
+                            "id": str(installment["id"]),
+                            "amount": float(installment["amount"]),
+                            "currency": settings.DEFAULT_CURRENCY,
+                            "due_date": format_date(installment["due_date"]),
+                            "state": installment["state"],
+                        }
+                        for installment in order.payment_schedule
+                    ]
+                    if order.payment_schedule
+                    else None,
```

Comment:
Are those two lines really useful in this test case ? 
Since we call `order.generate_schedule()` on line 767.